### ISSUE TITLE
prevPage fetching results from the start of the dataset

### ISSUE THREAD

**Issue by DrFacepalm** [MatrixAI/js-pagination#11]
The cursor prevPage sets order to false, which is not enough to return things in the correct order. When the backend sets order to false,
it does the following:
```
Data:                   Seek
-------------------------|--------
-results-->
```

What needs to happen is the following:
```
Data:                   Seek
-------------------------|--------
               <-results--
```
Which is achieved by setting `sort` to `desc`.

**Comment by CMCDragonkai** [comment]
So it turns out `order: true` is just the "default" direction of pagination. And `order: false` is just the opposite direction of pagination.

Now when `order: false`, the problem is that for APIs, we have another parameter called `sort` which can actually specify what the sort of the data should be.

I think to make this foolproof we have to record the `sort` as well.

Then when we go previous, we are definitely going `order: false`, but we have to flip what the `sort` is.

But what is the default sort? It has to be user specified, specifically on `pageInit`. There are 2 choices: `asc` and `desc`. Whatever the choice is, we must flip that choice when we flip the `order`.

**Comment by DrFacepalm** [comment]
Another thing to consider is that if we say order is true, we really mean that we use the default ordering. `order` doesn't actually order anything, it just refers to "Items before or after the seek", with "Before" and "After" being relative to the default direction.

If we tie `order=true` to `sort=asc` and `order=false` to `sort=false` then this will only be correct under the assumption that the default direction is ascending, which may not always be the case. This sounds to me like setting `sort=asc/desc` should be done as part of the function that `prevPage` takes, (or something along those lines).

**Comment by CMCDragonkai** [comment]
> **order doesn't actually order anything, it just refers to "Items before or after the seek", with "Before" and "After" being relative to the default direction.**

**Comment by CMCDragonkai** [comment]
This is the important thing!! Have to fix this and release `1.1.0` for js-pagination since its a MAJOR feature lol.

**Comment by CMCDragonkai** [comment]
So what's happening right now is that with:

```
['A', 'B', 'C', 'D', 'E']
```

Right now when `order: false` all we are doing is flipping the comparison in the database.

We are just flipping between:

```
-- for order == true
WHERE created_on > seek
```

```
-- for order == false
WHERE created_on < seek
```

But from a database approach, this is not enough. Because this would just get results from the beginning. That is the left of the entire data stream.

So if we want the order to actually mean to flip the order of the seek, and to record the limit from there, how would we do this?
